e2e/cypress/integration/enterprise/cloud/onboarding/next_steps_tips_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @onboarding @cloud_only
+
+describe('Cloud Onboarding - Next Steps Tips', () => {
```

Comment:
To match the header shown and test case
```suggestion
describe('Cloud Onboarding - Tips and Next Steps', () => {
```